2-projects/1-todo-list/script.js

```diff
@@ -17,7 +37,12 @@ function addNewTodo(event) {
   // The code below prevents the page from refreshing when we click the 'Add Todo' button.
   event.preventDefault();
   // Write your code here... and remember to reset the input field to be blank after creating a todo!
+  let inputField = document.getElementById("todoInput");
+  appendToDo({ task: inputField.value, completed: false });
+  console.log("button");
 }
+let toDoBtn = document.getElementById("add-btn");
+toDoBtn.addEventListener("click", addNewTodo);
```

Comment:
This is all really good, a nice improvement would be to add the new todo to the todos array and the re-render the array. this would allow you to save the todos array to somewhere like local storage so they can be repopulated when a user comes back later